### PR TITLE
Add Functionality

### PR DESCRIPTION
**Pull request by aeh1707**
Add basic CRUDs functionality, using a modular approach, plus a native storage capabality.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,40 +1,96 @@
+/* eslint-disable */
```
Comment (zieeco) [inline src/index.js]: **[REQUIRED CHANGES]**

- Please, let's try to write clean codes that conform to the linter. The usage of linters would help us as developers to write better code. Please, kindly remove it and try to fix the linter error.

File: src/modules/CRUD.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
```
Comment (zieeco) [inline src/modules/CRUD.js]: **[REQUIRED CHANGES]**

- Please, let's try to write clean codes that conform to the linter. The usage of linters would help us as developers to write better code. Please, kindly remove it and try to fix the linter error.

File: src/modules/CRUD.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  }
+};
+
+export const removeAll = () => {
+  tasks = [];
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+export const addTask = (taskDescription) => {
+  const newTask = new taskObject(taskDescription, false, tasks.length + 1);
```
Comment (zieeco) [inline src/modules/CRUD.js]: **[REQUIRED CHANGES]**

- Please, kindly ensure your app is working as a user can't add a task. here's the error from the console below:

ensure

**Review by zieeco — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES :recycle: 

Hi, @aeh1707, 

Good job so far!
However, you still need to work on some issues to go to the next project, but you are almost there!

## Highlights

- Nice Design :heavy_check_mark: 


## Required Changes ♻️

- [ ] Please, kindly add the `getting started` header in your readme to tell the user how to have your project up and running on their machine to make your readme more professional.
- [ ] Also, kindly rename the `Authors` to `Author` as you've done the project alone.

_Check the comments under the review._




## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @aeh1707 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**